.github/workflows/publish-toolkit.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         fi
 
     - name: Send status to Slack
-      if: steps.Publish_Toolkit.outputs.skip_publish != 'true'
+      if: always() && steps.Publish_Toolkit.outputs.skip_publish != 'true'
```

Comment:
need to add always() so that this step will run even when the previous step has a non-zero exit code